### DISCUSSION TITLE
References starting from [0] with possible solution

### DISCUSSION THREAD

**Discussion by simpleParadox** [draldric/uAlberta-Thesis-LaTeX-Template#2] [General]
Hello,
The references in the bibliography start from [0] instead of [1] for the settings provided in the ualberta.cls file.
After a lot of trial and error, it looks like the 'defernumbers=true' attribute in the \usepackage[biblatex] statement is causing this to happen. After removing the attribute, everything seems to work fine and references start from [1].

I usually haven't seen references start from [0], was this included for a specific purpose? I'm assuming it should affect anything else in the .tex but thought of posting it.

**Comment by draldric** [comment]
Hello,
Thank you for your suggestion.
The reference were not designed this way, this is a result of changing the bib file, such as adding more references to the bib file.
The attribute is included for those who have bibliographies at the end of chapters, such as in the case of an embedded publication or for stylistic reasons. Because FGSR requires a single master reference list, this means that there could be multiple places with the same references (smaller ones and the large one), this attribute addresses this.

The actual solution to the problem is addressed by the biblatex package: "An important thing to note is that if you are using this option, then changes to options, the bib file or certain commands like \printbibliography will usually require that you delete your current aux file and re-run LaTeX to obtain the correct numbering."
